src/orchestrate.scm

```diff
@@ -233,24 +233,19 @@
 			(List (Variable "sentence")))
 	))
 
-; Show happy emotion.
-; XXX FIXME -- these have hard-coded length-of-time values in them.
-; Most other similar behaviors have randomized values, which are
-; controlled by bounds in the config files - cfg-sophia and cfg-eva.scm
+; for salient
 (DefineLink
-    (DefinedPredicate "Quiet:happy")
-    (Evaluation
-        (GroundedPredicate "py: do_emotion")
-        (List (Concept "happy") (NumberNode 3) (NumberNode 0.5))
-    ))
-
-; Show amused emotion
-(DefineLink
-    (DefinedPredicate "Normal:amused")
-    (Evaluation
-        (GroundedPredicate "py: do_emotion")
-        (List (Concept "amused") (NumberNode 3) (NumberNode 0.5))
-    ))
+    (DefinedPredicate "look at salient point")
+    (AndLink
```

Comment:
audioSource.py (found under sound directory in HEAD) publishes to /opencog/AudioFeature and /opencog/suddenchange topics